server/src/main/java/io/crate/rest/action/SqlHttpHandler.java

```diff
@@ -199,8 +202,8 @@ private void sendResponse(Session session,
         List<Object> args = parseContext.args();
         List<List<Object>> bulkArgs = parseContext.bulkArgs();
         if (bothProvided(args, bulkArgs)) {
-            return CompletableFuture.failedFuture(new SQLActionException(
-                "request body contains args and bulk_args. It's forbidden to provide both", 4000, HttpResponseStatus.BAD_REQUEST));
+            return CompletableFuture.failedFuture(new InvalidRequest(
```

Comment:
`InvalidRequest` is only introduced to handle this case here, is there a simpler way to do this ?

Reply:
`IllegalArgumentException` wouldn't fit?